src/longbow/utils/bam_utils.py

```diff
@@ -418,6 +418,11 @@ def get_model_name_from_bam_header(header):
     return get_model_name_from_bam_header(header)['name']
 
 
+def get_model_from_bam_header(header):
+    model_jsons = get_models_from_bam_header(header)
+    return model_jsons[0]
```

Comment:
Can you add a warning or error if there is more than one model in the header?

Reply:
Will do.